setup.py

```diff
@@ -312,62 +312,47 @@ def build_extensions(self):
 
         elif sys.platform.startswith("linux"):
             arch_tp = (plat.processor(), plat.architecture()[0])
-            if arch_tp == ("x86_64", "32bit"):
-                # 32-bit build on 64-bit machine.
+            # This should be correct on debian derivatives.
+            if plat.dist()[0].lower() in ('debian', 'ubuntu'):
```

Comment:
`platform.dist()` is deprecated and will be removed in Python 3.7

Reply:
Great. Some batteries included I guess.